web/pdf_page_view.js

```diff
@@ -1006,20 +980,27 @@ class PDFPageView {
 
     // Keep the canvas hidden until the first draw callback, or until drawing
     // is complete when `!this.renderingQueue`, to prevent black flickering.
```

Comment:
This comments needs to be updated, since we are not keeping the canvas hidden until the first draw callback anymore (at least, not when we are re-drawing).